.github/workflows/preview.yml

```diff
@@ -23,12 +23,11 @@ jobs:
       - name: Restore mtime
         run: git restore-mtime
 
-      # There is a weird issue with download-artifact@v4
-      # keeping the external action for now
```

Comment:
Please keep that comment.
It's about the GitHub-provided action not working as expected as in theory the official action should be able to handle this case now.

Reply:
Ah, ok, will do! I stared at the comment for a while trying to decide if it really meant `download-artifact@v4`, or if it meant `actions-download-artifact@v4`, and someone had just used a shorthand in the comment. Since `actions-download-artifact` did have some weird issues in v4, that muddied the waters. 

Darn, that means my change isn't reducing the line count. :)

Reply:
Yeah, the full story is that I want to get rid of all the actions that are not either official or ours.

But I haven't found a way to get the official one work in this case, even if the new version was supposed to cover this use case.

Reply:
Ah... I don't think the requirement of adding the token was documented before (and the message when it didn't work wasn't very clear)... We might want to test it again with the token added.
I'll test it on the website and see how it goes.